README.md

```diff
@@ -169,6 +169,11 @@ reasons:
 2. Knowing that a commit has been tested, which is e.g. important for bisection
    so you can easily and automatically `git bisect --skip` untested commits.
 
+Additionally, by using `--add-part-of`, all commits will be tagged with a link
```

Comment:
If you want to use "tagged" I'd say "commit message" rather than commit to disambiguate.